packages/truffle-compile-vyper/index.js

```diff
@@ -0,0 +1,175 @@
+const path = require('path');
+const exec = require('child_process').exec;
+
+const async = require('async');
+const colors = require('colors');
+const minimatch = require('minimatch');
+
+const find_contracts = require('truffle-contract-sources');
+const Profiler = require('truffle-compile/profiler');
+
+const compiler = {
+  name: 'vyper',
+  version: null,
+};
+
+const VYPER_PATTERN = '**/*.vy';
```

Comment:
Can we have `v.py` and `vyper.py` as patterns too? A lot of people keep them in `.py` for syntax highlighting.

Reply:
Makes sense, will add support for these.

Reply:
Added support for `.v.py` and `.vyper.py` files.